src/Test.UnitTests.Sarif/FileRegionsCacheTests.cs

```diff
@@ -745,8 +740,7 @@ public void FileRegionsCache_PopulatesWithOneLine_IncreasingToTheRight()
             region = fileRegionsCache.PopulateTextRegionProperties(region, uri, true, content);
             Region multilineRegion = fileRegionsCache.ConstructMultilineContextSnippet(region, uri);
 
-            // CharLength + 128 to the right = 428 characters
-            multilineRegion.CharLength.Should().Be(300 + 128);
+            multilineRegion.CharLength.Should().Be(512);
```

Comment:
And shouldn't that be  a constant?

Reply:
+1 to Craig's comment. We should have an `Assume` section in the test defining the constants.

Reply:
We now pick it up from the product, good call.